trade-finance-manager-api/src/v1/api.js

```diff
@@ -523,21 +523,18 @@ const getFunctionsAPI = async (type = CONSTANTS.DURABLE_FUNCTIONS.TYPE.ACBS, url
   }
 };
 
-const createEstoreFolders = async (eStoreFolderInfo) => {
+const createEstoreFolders = async (data) => {
   try {
-    const response = await axios({
+    await axios({
       method: 'post',
       url: `${refDataUrl}/estore`,
       headers: {
         'Content-Type': 'application/json',
       },
-      data: {
-        eStoreFolderInfo,
-      },
+      data,
     });
-    return response.data;
   } catch (err) {
-    return {};
+    console.error(err);
```

Comment:
Can you please output `{ err }`, helps in debugging. Thanks

Reply:
sorted